config.schema.json

```diff
@@ -89,6 +89,21 @@
         "title": "Light",
         "type": "boolean",
         "default": true
+      },
+      "turboswitch": {
+        "title": "Turbo Switch",
```

Comment:
I wonder if this should be changed to "Enable Turbo Switch Homekit service", to make it more descriptive. Same goes for the one below. (Open for other suggestions)